### PR TITLE
AutoDesk.DWGTrueView version 4.0.19.0

### PR DESCRIPTION
**Pull request by oliveirahsi**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59692)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I'm stuck here:
image

File: manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: AutoDesk.DWGTrueView
+PackageVersion: 4.0.19.0
+InstallerSwitches:
+  Silent: --silent
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://efulfillment.autodesk.com/NetSWDLD/2023/ACD/EC2FCD9E-AC4D-3E9D-8526-96832A231455/SFX/DWGTrueView_2023_English_64bit_dlm.sfx.exe?authparam=b91ba1b191a933c9c10b81c98913132b812b1b811b991b3389b98913a1c933b9&SESSION_ID=123456789;1083219296;2356937597;1;authparam;SESSION_ID
```
Comment (Trenly) [inline manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.installer.yaml:11]: The URL contains an authparam and a sessionID, which will likely expire and block future installations

File: manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: AutoDesk.DWGTrueView
+PackageVersion: 4.0.19.0
+InstallerSwitches:
+  Silent: --silent
```
Comment (Trenly) [inline manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.installer.yaml:7]: These switches do not appear accurate

File: manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: AutoDesk.DWGTrueView
+PackageVersion: 4.0.19.0
+PackageLocale: en-US
+Publisher: Autodesk, Inc.
+PackageName: Autodesk DWG TrueView
+License: Free
```
Comment (Trenly) [inline manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```
Autodesk has a EULA which is part of the install process, this is not freeware

File: manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: AutoDesk.DWGTrueView
```
Comment (Trenly) [inline manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.installer.yaml:4]:
```suggestion
PackageIdentifier: Autodesk.DWGTrueView
```
Publisher already exists as `Autodesk` - Note the capitalization

File: manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: AutoDesk.DWGTrueView
```
Comment (Trenly) [inline manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Autodesk.DWGTrueView
```
Publisher already exists as `Autodesk` - Note the capitalization

File: manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: AutoDesk.DWGTrueView
```
Comment (Trenly) [inline manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.yaml:4]:
```suggestion
PackageIdentifier: Autodesk.DWGTrueView
```
Publisher already exists as `Autodesk` - Note the capitalization

File: manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: AutoDesk.DWGTrueView
+PackageVersion: 4.0.19.0
```
Comment (Trenly) [inline manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.installer.yaml:5]:
```suggestion
PackageVersion: 24.2.72.0
```
Version must match control panel

File: manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: AutoDesk.DWGTrueView
+PackageVersion: 4.0.19.0
```
Comment (Trenly) [inline manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 24.2.72.0
```
Version must match control panel

File: manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.yaml [hunk 9]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: AutoDesk.DWGTrueView
+PackageVersion: 4.0.19.0
```
Comment (Trenly) [inline manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.yaml:5]:
```suggestion
PackageVersion: 24.2.72.0
```
Version must match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Pull request closed without merging**